content_schemas/formats/how_government_works.jsonnet

```diff
@@ -0,0 +1,73 @@
+(import "shared/default_format.jsonnet") + {
+  document_type: [
+    "how_government_works"
+  ],
+  definitions: {
+    details: {
+      type: "object",
+      additionalProperties: false,
```

Comment:
Do we want to add the reshuffle mode as a required field to this details hash?

Reply:
Good idea 👍